fabtests/benchmarks/benchmark_shared.c

```diff
@@ -460,17 +486,25 @@ int bandwidth_rma(enum ft_rma_opcodes rma_op, struct fi_rma_iov *remote)
 							tx_buf + offset,
 							opts.transfer_size,
 							remote);
```

Comment:
Inject and fi_more are not excluded. I think the logic is a bit misleading here. `opts.use_fi_more` shouldn't be an `else if` to `inject`. Instead, it's better to make `if (opts.use_fi_more)` inside the `else` block.

Reply:
I don't see there is a difference here between `if ... else if ... else ...` and `if ... else { if ... else ... }`. The current form is simpler.